src/cli/repl/commands/repl-cfg.ts

```diff
@@ -44,8 +44,8 @@ export const controlflowCommand: ReplCommand = {
 	usageExample: ':controlflow',
 	aliases:      [ 'cfg', 'cf' ],
 	script:       false,
-	fn:           async(output, shell, remainingLine) => {
-		await produceAndPrintCfg(shell, remainingLine, output, [], cfgToMermaid);
+	fn:           async(output, shell, remainingLine, _, config) => {
```

Comment:
Given that this now gets longer and longer we should transition to an object based passing style so that we can select the keys of interest! What do you say @MaxAtoms?

Reply:
Sound good :) I added this with 6d15b86c87902ce753c21f18c46f7209cd15d98a